conn.go

```diff
@@ -331,6 +332,28 @@ func (c *Conn) Close() error {
 	return c.conn.Close()
 }
 
+
+// Shutdown sends a close frame to the peer and waits for close frame in resopnse.
+// Shutdown assumes that the application is reading the connection in another
+// goroutine and hence it does not try to read close frame itself
```

Comment:
The wording is not strong enough here.  The documentation should say "The application must not call Shutdown from the reading goroutine."